cc3d/core/RollbackImporter.py

```diff
@@ -70,7 +70,10 @@ def _import(self, name, globals=None, locals=None, fromlist=[], level=0):
 
```

Comment:
We need to figure out how to structure the code so that we don't have to use RollbackImporter. currently it unloaded all swig modules and allows us to run multiple simulations without closing the player but maybe there is a better way. RollbackImporter is a bit complicated

Reply:
Agreed. A service architecture like what we're doing in [feature/jupyter](https://github.com/CompuCell3D/CompuCell3D/tree/feature/jupyter) might do the trick.